api/server/main.ts

```diff
@@ -1,54 +1,87 @@
 import { Meteor } from 'meteor/meteor';
 import { Accounts } from 'meteor/accounts-base';
 import { Users } from './collections/users';
+import { Roles } from 'meteor/alanning:roles';
+
+import * as _ from 'lodash';
+
+
 
 Meteor.startup(() => {
+
+  //VARIABLES
+  //BE SURE TO SET MAIL_RUL
+  //process.env.MAIL_URL
+
   if (Meteor.settings) {
     Object.assign(Accounts._options, Meteor.settings['accounts-phone']);
-    SMS.twilio = Meteor.settings['twilio'];
   }
 
-  if (Users.collection.find().count() > 0) {
-    return;
+  //UNCOMMENT AND RUN TO CLEAR ALL USERS
+  Users.remove({});
```

Comment:
so I guess we just need to re-comment this at some point before merging next PR?

Reply:
Yes, that is utility code for development, should probably not go into the PR. I'll update.